src/Parser/Core/Modules/SpellHistory.js

```diff
@@ -81,7 +81,8 @@ class SpellHistory extends Analyzer {
   }
   on_toPlayer_updatespellusable(event) {
     const spellId = event.ability.guid;
-    this._append(spellId, this._addTimeWaitingOnGCD(event));
+    const extendedEvent = this._eventWithTimeWaitingOnGCD(event);
```

Comment:
I meant to just have it on a separate line. Returning the event suggests the returned value is different, but it's not since objects are passed by reference and the original object is mutated.

I'll change it quickly to what I meant so this can be merged.